sstables/sstable_set.cc

```diff
@@ -820,7 +820,7 @@ class incremental_reader_selector : public reader_selector {
         auto readers = std::vector<flat_mutation_reader_v2>();
 
         do {
-            auto selection = _selector->select(_selector_position);
+            auto selection = _selector->select({_selector_position, _pr});
```

Comment:
Ah, we already sent _pr via create_reader(), but the next tablet's sstables would just reject it.